### PR TITLE
Implemented versioning dependency injection and clean service registration

### PR DESCRIPTION
**Pull request by MariaCrucin**

### CODE REVIEW

File: TweetBook/Program.cs [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
-// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
-builder.Services.AddEndpointsApiExplorer();
-builder.Services.AddSwaggerGen();
+builder.InstallServicesInAssembly();
+
+//builder.Services.AddDbContext<DataContext>(options => options.UseSqlite(connectionString));
+
+//builder.Services.AddControllers();
+//// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+//builder.Services.AddEndpointsApiExplorer();
```
Comment (SebiBarson) [inline TweetBook/Program.cs]: please refrain from commenting code

**Review by SebiBarson — CHANGES_REQUESTED** [review]

File: TweetBook/Program.cs [hunk 2]
```diff
@@ -1,11 +1,21 @@
+
+using Microsoft.EntityFrameworkCore;
+using TweetBook.Data;
+using TweetBook.Installers;
+
 var builder = WebApplication.CreateBuilder(args);
+//var connectionString = builder.Configuration.GetConnectionString("DefaultConnection");
```
Comment (SebiBarson) [inline TweetBook/Program.cs]: please refrain from commenting code

**Review by SebiBarson — CHANGES_REQUESTED** [review]

File: TweetBook/Program.cs (hunk 1 above)
Comment (MariaCrucin) [reply TweetBook/Program.cs]: Done

File: TweetBook/Program.cs (hunk 2 above)
Comment (MariaCrucin) [reply TweetBook/Program.cs]: Done

**Review by SebiBarson — APPROVED** [review]

**Pull request merged**